docs/pages/access-controls/reference.mdx

```diff
@@ -357,9 +357,38 @@ allow:
     - resources:
         - token
       verbs: [list, create, read, update, delete]
+```
+
+### Allowing access to token resources
+
+You should note that if you configure a role that allows tokens to be created, a user assigned to the 
+role can create any type of token. 
+For example, you might create a role with the following configuration to enable assigned
+users to enroll servers:
 
+```yaml
+kind: role
+version: v7
+metadata:
+  name: enroll-servers
+spec:
+  allow:
+    node_labels:
+      'env': 'us-lab'
+    rules:
+      - resources: [token]
+        verbs: [list, create, read, update, delete]
+  deny: {}
 ```
 
+With these permissions, users assigned this role can generate any type of token, including a token to enroll 
+a server, application, or database, establish a trust relationship between a root cluster and a new Teleport 
+Proxy Service or leaf cluster, or configure a device trust relationship for a managed device.
+Because the token resource isn't scoped to a specific context, such as a node or trusted cluster, you should 
```

Comment:
```suggestion

Because the token resource isn't scoped to a specific context, such as a node or trusted cluster, you should 
```
To break up a long paragraph